required/latex-lab/latex-lab-footnotes.dtx

```diff
@@ -1854,13 +1853,13 @@
          {
            \exp_not:N\fnote_gput_refs:ee
             { \tag_get:n{struct_num} }
-            { \ref_value:Vn\l_@@_currentlabel_tl {fnote/struct} }
+            { \property_ref:ee{ @@/\l_@@_currentlabel_tl } {fnote/struct} }
```

Comment:
:ee doesn't seem to make sense here and only costs processing cycles, should be :en I guess

Reply:
there is no `\property_ref:en`, so the question is if one should generate a variant for this?

Reply:
I think yes. After all executing "e" on somehting that is already a fixed string is just costing execution time.